sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcUtils.scala

```diff
@@ -110,4 +107,23 @@ object OrcUtils extends Logging {
       }
     }
   }
+
+  /**
+   * Return a fixed ORC schema with data schema information, if needed.
+   * The schema inside old ORC files might consist of invalid column names like '_col0'.
+   */
+  def getFixedTypeDescription(
```

Comment:
Do we really need this? The ORC schema is used to create the ORC batch, for the batch data I think we only care about the data types not field names.

Reply:
Okay. I'll remove this from this PR.